src/frontend/packages/lib_markdown/src/components/CodeMirrorEditor/index.tsx

```diff
@@ -1,12 +1,13 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
Why is it needed and can't be properly fix ?

Reply:
I took shorcuts with eslint rules, but I'll try to fix them.

Reply:
Oh sorry this one, once fixed accordingly to eslint, raises a annoying bug: 
As the useEffect dependencies should be `codemirrorEditor, initialContent, onEditorContentChange` when we press a key, the editor is rendered again, and we lose the focus, preventing another key stroke.